packages/web/src/application/test/zoom/zoom-provider.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactNode } from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import { User } from '@chess-tent/models';
+import application from '@application';
+import { users, zoom } from '../fixtures';
+import { getPasswordInput, getMeetingNumberInput } from './utils';
+
+const { components } = application;
+
+interface RenderOptions {
+  user: User;
+  meetingNumber?: string;
+  queryCode?: boolean;
+}
+
+const renderWithProvider = (children: ReactNode, options: RenderOptions) => {
+  const { ZoomProvider } = components;
```

Comment:
One extra thing. 

ZoomProvider should be in the providers folder inside Zoom module. Not components.

Reply:
What do you mean? Should I define it differently than it is?

Reply:
Then a new type `providers` should be created application wide, and also other Providers moved there?

Reply:
I see now on the others, should I remove ZoomProvider from components and define it as for example in user/providers ?

Reply:
Not in this case. Zoom provider is scoped, `addProvider` is for global providers.

Ideally there should be a namespace `providers` but let's leave that for now. 

Just move zoom provider as I suggested in a different folder, leave the rest.